requirements.txt

```diff
@@ -1 +1,2 @@
 apache-airflow == 2.5.3
+requests >= 2.28.2
```

Comment:
is this needed after airflow version upgrade and not before

Reply:
This is just right practice to maintain requirements.txt updated for a python project. It already was using it, just that we never updated it I think.

Reply:
we import requests in the test code. so, added it. requests is also a dependency of airflow. Since, we import it, we better have it in this file